src/js/view/view.js

```diff
@@ -417,12 +417,15 @@ define([
 
         function itemReady(item) {
             const provider = _model.getVideo();
```

Comment:
Since we're checking for the presence of a videotag with `_videoLayer`, we no longer need the provider in this function.